### PR TITLE
navigo: update lastRouteResolved

### PR DESCRIPTION
**Pull request by deini**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Navigo](https://github.com/krasimir/navigo/blob/4f7315525eb3dce4912a206c4ade90bf27f63a18/src/index.js#L261)

I'm not sure how to test it, there was no `lastRouteResolved` test, would love some guidance on this.

### CODE REVIEW

File: types/navigo/index.d.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // Project: https://github.com/krasimir/navigo
 // Definitions by: Adrian Ehrsam <https://github.com/aersamkull>
 //                 Dancespiele <https://github.com/dancespiele>
+//                 Daniel Almaguer <https://github.com/deini>
```
Comment (dancespiele) [inline types/navigo/index.d.ts]: We are already two reviewers. Do you need to be reviewer too?. Waiting what think @aersamkull about it

**Review by spielcrypto — CHANGES_REQUESTED** [review]

File: types/navigo/index.d.ts (hunk 1 above)
Comment (deini) [reply types/navigo/index.d.ts]: @dancespiele I don't need to be a reviewer, but I don't see why not. The more the better? 

Unless I'm missing something, if there are more reviewers wouldn't it make it easier to spot discrepancies on the typings such as the ones I'm updating?

Comment (aersamkull) [reply types/navigo/index.d.ts]: Why not, the more the better :)

**Review by aersam — APPROVED** [review]

**Review by spielcrypto — APPROVED** [review]

File: types/navigo/index.d.ts (hunk 1 above)
Comment (dancespiele) [reply types/navigo/index.d.ts]: Ok then I approve

**Comment by dancespiele** [comment]
@deini about this "I'm not sure how to test it, there was no lastRouteResolved test, would love some guidance on this." I tested like this
```typescript
it("has to get the last route resolved", (done) => {
      router.navigate("/child/5");
      setTimeout(() => {
        expect(router.lastRouteResolved().url).has.to.be.equal("/child/5");
        done();
      });
});
```
I don't know it help you

**Pull request merged**